src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/EETypeNode.GCDesc.cs

```diff
@@ -0,0 +1,225 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+using Internal.TypeSystem;
+
+using Debug = System.Diagnostics.Debug;
+
+namespace ILCompiler.DependencyAnalysis
+{
+    partial class EETypeNode
+    {
+        private ISymbolNode _gcdesc;
+
+        private ISymbolNode GCDesc
+        {
+            get
+            {
+                if (GCDescSize == 0)
```

Comment:
This symbol is not really needed by anything, but not defining a symbol at offset 0 of a Node trips up the object writer. This might be an ObjectWriter bug, but I didn't try to understand what's going on yet.

For reference, the line that gets tripped up is:

```csharp
// First entry is the node (entry point) name.
_currentNodeName = _offsetToDefName[0][0];
```